ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -790,14 +795,30 @@ public void setEnabled(boolean enabled)
       }
       else
       {
-         streamingDisabled.poll();
          sleepToolbox();
-         visualizeIKPreviewGraphic(true);
-         streamToController.set(false);
+         setStreamingEnabled(false, false);
+      }
+
+      previewEnabled.set(enabled);
+   }
+
+   public void setStreamingEnabled(boolean enabled, boolean changed)
```

Comment:
Needed the changed boolean to keep the Control/Stop Robot checkbox, since it changes the value itself